src/client/index.ts

```diff
@@ -89,20 +125,25 @@ class Hammerhead {
         };
 
         this.PROCESSING_COMMENTS = {
-            stylesheetStart: StyleProcessor.STYLESHEET_PROCESSING_START_COMMENT,
-            stylesheetEnd:   StyleProcessor.STYLESHEET_PROCESSING_END_COMMENT,
+            stylesheetStart: styleProcessor.STYLESHEET_PROCESSING_START_COMMENT,
```

Comment:
Add a an additional namespace:
```js
this.PROCESSING_COMMENTS.stylesheet.start
this.PROCESSING_COMMENTS.stylesheet.end
this.PROCESSING_COMMENTS.script.start
...
```

Reply:
Now `PROCESSING_COMMENTS` is not used anywhere in the code, so I just removed it.